tests/test-sources/plugins/bufferlines/navic.nix

```diff
@@ -0,0 +1,16 @@
+{
+  empty = {
+    plugins.navic.enable = true;
+  };
+
+  defaults = {
+    plugins.navic = {
+      enable = true;
+      lsp = {
+        autoAttach = true;
+      };
+      separator = " | ";
+      click = true;
+    };
+  };
```

Comment:
Please add all the defined plugin options here.

Reply:
does that include every icon :sweat:  or is one good enough as an example ?

Reply:
Yes, I would say that you copy-paste the default from the plugin option declaration.